vendor/openvino/openvino.BUILD

```diff
@@ -8,378 +8,36 @@ TAGS = [
-)
-
-cc_library(
-    name = "ngraph",
-    srcs = glob(
-        [
-            "ngraph/core/builder/src/*.cpp",
-            "ngraph/core/builder/src/builder/*.cpp",
-            "ngraph/core/reference/src/*.cpp",
-            "ngraph/core/reference/src/runtime/**/*.cpp",
-            "ngraph/core/src/descriptor/*.cpp",
-            "ngraph/core/src/**/*.cpp",
-            "ngraph/core/src/*.cpp",
-        ],
-        exclude = [
-            "ngraph/src/ngraph/serializer.cpp",
-        ],
-    ),
-    hdrs = glob(
-        [
-            "ngraph/core/builder/include/ngraph/builder/*.hpp",
-            "ngraph/core/include/ngraph/*.hpp",
-            "ngraph/core/include/ngraph/**/*.hpp",
-            "ngraph/core/reference/include/ngraph/runtime/reference/*.hpp",
-            "ngraph/core/src/*.hpp",
-        ],
-    ),
-    copts = ["-w"],
-    defines = [
-        "NGRAPH_JSON_DISABLE",
-        "NGRAPH_INTERPRETER_ENABLE",
-        "NGRAPH_VERSION=\\\"0.21.0\\\"",
-    ],
-    includes = [
-        "ngraph/core/builder/include",
-        "ngraph/core/builder/include/ngraph",
-        "ngraph/core/builder/include/ngraph/builder",
-        "ngraph/core/include",
-        "ngraph/core/include/ngraph",
-        "ngraph/core/include/ngraph/op",
-        "ngraph/core/include/ngraph/op/util",
-        "ngraph/core/include/ngraph/pass",
-        "ngraph/core/include/ngraph/pattern",
-        "ngraph/core/include/ngraph/pattern/op",
-        "ngraph/core/include/ngraph/runtime",
-        "ngraph/core/reference/include",
-        "ngraph/core/src",
-    ],
-    local_defines = [
-        "PROJECT_ROOT_DIR=\\\"./\\\"",
-        "SHARED_LIB_PREFIX=\\\"\\\"",
-        "SHARED_LIB_SUFFIX=\\\"\\\"",
-    ],
-    tags = TAGS,
-    deps = [
-        ":itt",
-        "@tbb",
-    ],
-)
-
-cc_library(
-    name = "ngraph_tests",
-    srcs = glob([
-        "ngraph/test/runtime/*.cpp",
-        "ngraph/test/runtime/dynamic/*.cpp",
-        "ngraph/test/runtime/interpreter/*.cpp",
-        "ngraph/test/runtime/pass/*.cpp",
-        "ngraph/test/runtime/op/*.cpp",
-    ]),
-    hdrs = glob([
-        "ngraph/test/runtime/*.hpp",
-        "ngraph/test/runtime/dynamic/*.hpp",
-        "ngraph/test/runtime/interpreter/*.hpp",
-        "ngraph/test/runtime/pass/*.hpp",
-        "ngraph/test/runtime/op/*.hpp",
-    ]),
-    copts = ["-w"],
-    includes = [
-        "ngraph/test/runtime",
-    ],
-    local_defines = [
-        "PROJECT_ROOT_DIR=\\\"./\\\"",
-        "SHARED_LIB_PREFIX=\\\"\\\"",
-        "SHARED_LIB_SUFFIX=\\\"\\\"",
-    ],
-    deps = [
-        ":ngraph",
-    ],
+filegroup(
+    name = "all",
+    srcs = glob(["**"]),
+    visibility = ["//visibility:public"],
+)
+
+genrule(
+    name = "benchmarkapp",
+    srcs = [
+        ":all",
+        "@//plaidml:sdk_tar",
+    ],
+    outs = [
+        "benchmark_app",
+        "libtbb.so.2",
+        "libplaidml.so",
+        "libMKLDNNPlugin.so",
+        "libPlaidMLPlugin.so",
+    ],
+    cmd = """
```

Comment:
I'd prefer this was a python script, something that's much easier to maintain

Reply:
agreed but you only really get to inject a BUILD file - I could try and add additional patches or we could commit the script to the ov repo, but that feels a bit odd